akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -122,10 +122,20 @@ class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerW
     val builder = ContainerFormats.ActorInitializationException.newBuilder()
     if (ex.getActor ne null)
       builder.setActor(actorRefBuilder(ex.getActor))
-
     builder.setMessage(ex.getMessage).setCause(payloadSupport.payloadBuilder(ex.getCause)).build().toByteArray
   }
 
+  private def serializeThrowableNotSerializableException(ex: ThrowableNotSerializableException): Array[Byte] = {
```

Comment:
the `ThrowableNotSerializableException` should be backported to 2.5.x to support rolling update